internal/cmd/workspaces.go

```diff
@@ -489,9 +506,13 @@ coder workspaces create my-new-powerful-workspace --cpu 12 --disk 100 --memory 1
 				return nil
 			}
 
+			extraFlags := ""
+			if forEmail != coder.Me && forEmail != "" {
+				extraFlags = " --user " + forEmail
+			}
 			clog.LogSuccess("creating workspace...",
 				clog.BlankLine,
-				clog.Tipf(`run "coder workspaces watch-build %s" to trail the build logs`, workspace.Name),
+				clog.Tipf(`run "coder workspaces watch-build %s%s" to trail the build logs`, workspace.Name, extraFlags),
```

Comment:
Does this work?

Reply:
yer.  She does

Reply:
https://github.com/cdr/coder-cli/blob/ccecc2f48f527ee9c54481ef8ad89d816ac20340/internal/cmd/rebuild.go#L184-L184